.gitignore

```diff
@@ -0,0 +1,9 @@
+# Virtual environment
+.venv
```

Comment:
So, this one is not using `virtualenv` but `venv`?

Reply:
Ah, I see below that you just named the folder `.venv` but actually use `virtualenv`.

Reply:
Correct. It uses `virtualenv` and I though just `.env` would be maybe to generic.